src/Symfony/Component/Cache/Adapter/CouchbaseBucketAdapter.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter;
+
+use Symfony\Component\Cache\Exception\CacheException;
+use Symfony\Component\Cache\Exception\InvalidArgumentException;
+use Symfony\Component\Cache\Marshaller\DefaultMarshaller;
+use Symfony\Component\Cache\Marshaller\MarshallerInterface;
+
+/**
+ * @author Antonio Jose Cerezo Aranda <aj.cerezo@gmail.com>
+ */
+class CouchbaseBucketAdapter extends AbstractAdapter
```

Comment:
can and shall we make this class final @nicolas-grekas ? Not sure right now how Symfony handles such things

Reply:
Hi @OskarStark,

From what I've seen none of the other adapters is defined as final class.